cmd/kube-proxy/app/server.go

```diff
@@ -252,7 +253,33 @@ func NewProxyServerDefault(config *ProxyServerConfig) (*ProxyServer, error) {
 		glog.V(2).Info("Using iptables Proxier.")
 		proxierIptables, err := iptables.NewProxier(iptInterface, execer, config.IptablesSyncPeriod, config.MasqueradeAll)
 		if err != nil {
-			glog.Fatalf("Unable to create proxier: %v", err)
+			iptables_err := err.Error()
+			iptables_err_str := ""
+			if strings.Contains(iptables_err, "iptables") {
+				//	Error is most probably caused by lack of iptables
+				// configuration
+				iptables_err_str = "Check if iptables has been enabled in the kernel. " +
```

Comment:
My point was that I'm not sure this error will be triggered in the event of those modules being missing.  If those modules are actually missing from the machine, I expect it to fail much later and much less obviously.

Do we have evidence (e.g. manual tests) that this error actually fires?  I don't see how it would.